patches/patch-sharpziplib-CA63358-TarBuffer.patch

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Cloud Software Group, Inc.
+# 
+# Redistribution and use in source and binary forms,
+# with or without modification, are permitted provided
+# that the following conditions are met:
+# 
+# *   Redistributions of source code must retain the above
+#     copyright notice, this list of conditions and the
+#     following disclaimer.
+# *   Redistributions in binary form must reproduce the above
+#     copyright notice, this list of conditions and the
+#     following disclaimer in the documentation and/or other
+#     materials provided with the distribution.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+# CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+# INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+# SUCH DAMAGE.
+
+--- a/SharpZipLib-1.3.3/src/ICSharpCode.SharpZipLib/Tar/TarBuffer.cs	2011-06-17 23:09:32.000000000 +0100
++++ b/SharpZipLib-1.3.3/src/ICSharpCode.SharpZipLib/Tar/TarBuffer.cs	2011-08-01 20:14:18.000000000 +0100
+@@ -286,55 +286,6 @@ namespace ICSharpCode.SharpZipLib.Tar
```

Comment:
How did we end up with code lines in the chunk headers? (Several more instances of it below).

Reply:
I'm assuming you mean the "namespace ICSharpCode.SharpZipLib.Tar"? If that's the case, it was auto generated by the git diff command. I tested with and without "namespace ICSharpCode.SharpZipLib.Tar", both are identical so happy to keep it so it is similar for future patches and also happy to remove it to keep the old convention.

Reply:
As long as the patch applies successfully it doesn't matter too much, but I think it is cleaner without it.

Reply:
Ok cool, yes both apply cleanly so will just remove it.